webrtc_to_hls/assets/src/app.ts

```diff
@@ -27,7 +27,7 @@ export const AUDIO_CONSTRAINTS: MediaStreamConstraints = {
 
 export const VIDEO_CONSTRAINTS: MediaStreamConstraints = {
   audio: false,
-  video: { width: 640, height: 360, frameRate: 24 },
+  video: { width: 1280, height: 720, frameRate: 24 },
```

Comment:
Why do we want to send HD video? This will take more bandwidth and CPU resources

Reply:
When I was developing this, somewhere I found information that HD is required for simulcast and only with HD constraints it was working. But as I am testing this now it can be decreased to previous constraints.

Reply:
Ok I remember on beginning I was trying to get 3 simulcast layers and based on this [discussion](https://groups.google.com/g/discuss-webrtc/c/czRbYBm1u2g/m/xgawiGCcDgAJ), I thought that HD is required. The correct limits are [here](https://chromium.googlesource.com/external/webrtc/+/master/media/engine/simulcast.cc#89).